src/__test__/Navigation.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router-dom';
+import Navigation from '../js/components/Navigation';
+
+jest.mock('../js/modules/radarService', () => ({
```

Comment:
I think it's cleaner to use the `__mocks__` pattern for large mocks like this, and makes it reusable (even though it's a bit overkill here perhaps)